style.css

```diff
@@ -0,0 +1,113 @@
+header {
+  display: flex;
+  background: #3c3a39;
+  justify-content: space-around;
+  position: absolute;
+  width: 100%;
+  height: 92px;
+  left: 0;
+  top: 0;
+  align-items: flex-end;
+}
+
+/* cyrillic */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: url(https://fonts.gstatic.com/s/roboto/v29/KFOmCnqEu92Fr1Mu5mxKOzY.woff2) format('woff2');
+  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
+}
+
+div {
+  display: flex;
+  flex-wrap: wrap;
+  color: #fff;
+  width: 30%;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  margin-left: -80px;
+  margin-bottom: 10px;
+}
+
+#burger-menu {
+  margin-right: -60px;
+  width: 18px;
+  height: 12px;
+  margin-bottom: 20px;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
```

Comment:
[OPTIONAL]

- Instead of directly styling selector, it would be better to give a proper class name and then style the class to use in different place according to your need.
```
in HTML

<div class="menu"> .... </div>

in CSS

.menu {
  display: flex;
  ...
  ...
}
```